### ISSUE TITLE
Samples Showing Xamarin Forms Integration?

### ISSUE THREAD

**Issue by aherrick** [protobuf-net/protobuf-net.Grpc#148]
Any samples showing XForms talking to an ASP Net Core 5 GRPC backend? Is this even possible?

I'm currently using JSON, but I'd like to move to a C# Code First approach using GRPC.

Thanks!

**Comment by mgravell** [comment]
Xamarin targeting what? It matters because of the AOT aspect, which I am working on via "generators"

**Comment by aherrick** [comment]
iOS and Android is really all I care about at this point. I'm just trying to see a New Xamarin Forms App talking to a Net 5 GRPC Core backend :)

I'd love to use GRPC instead of JSON on a new app I'm building.

**Comment by mgravell** [comment]
Right. I can't give you those until I have the generator code complete (or
at least, "complete enough"),due to the AOT aspect. Since this is hobby
work, it is dependent on my spare time availability.

**Comment by aherrick** [comment]
@mgravell understood! just figured I'd check. Thanks for your help :)
